test/data/README.md

```diff
@@ -46,6 +46,31 @@ Redistributing the dataset "taxi-fare-test.csv", "taxi-fare-train.csv" with attr
 > 
 > The dataset is provided under terms provided by City of New York: https://opendata.cityofnewyork.us/overview/#termsofuse.
 
+### MSLRWeb10k
```

Comment:
I don't understand why we need this. We are not distributing this data set with the repo (it isn't being checked in). So why would we list the citation here? Shouldn't the citation go where the dataset is actually being distributed?

We aren't doing it for the `winequality-white.csv` dataset that we are already downloading as part of the build.

Reply:
When downloading a specific file from the CDN, the user won't receive a secondary file, like the license file. We could (and perhaps should) maintain a secondary copy of citations/licenses in the CDN though a user wouldn't know it's there. Another possible location we can reference this, is the code where we download the CDN file. The MSLR-WEB10K/ MSLR-WEB30K datasets were previously released under a MSR non-commercial license. The license statement is needed to tell the world that MSFT is now licensing the full MSLR-WEB10K & MSLR-WEB30K datasets under a CC-by 4.0 license, and the citation here provides the citation required in the 'by' part of CC-by. In summary, we need the citation/license in some location. 

You're right, there's oddities in the way `winequality-white.csv` is distributed: https://github.com/dotnet/machinelearning/issues/889